content/docs/08-ws/00-overview.md

```diff
@@ -1,42 +1,45 @@
 
 The SpacetimeDB SDKs comminicate with their corresponding database using the WebSocket API.
 
 ## Connecting
 
-To initiate a WebSocket connection, send a `GET` request to the [`/database/subscribe/:name_or_address` endpoint](/docs/http/database#databasesubscribename_or_address-get) with headers appropriate to upgrade to a WebSocket connection as per [RFC 6455](https://datatracker.ietf.org/doc/html/rfc6455).
+To initiate a WebSocket connection, send a `GET` request to the [`/database/subscribe/:name_or_address` endpoint](/docs/http/database#database-subscribe-name-or-address-get) with headers appropriate to upgrade to a WebSocket connection as per [RFC 6455](https://datatracker.ietf.org/doc/html/rfc6455).
 
-To re-connect with an existing identity, include its token in a [SpacetimeDB Authorization header](/docs/http). Otherwise, a new identity and token will be generated for the client.
+To re-connect with an existing identity, include its token in a [SpacetimeDB Authorization header](/docs/http/overview#encoding-authorization-headers). Otherwise, a new identity and token will be generated for the client.
 
 ## Protocols
 
-Clients connecting via WebSocket can choose between two protocols, [`v1.bin.spacetimedb`](#binary-protocol) and [`v1.text.spacetimedb`](#text-protocol). Clients should include one of these protocols in the `Sec-WebSocket-Protocol` header of their request.
+Clients connecting via WebSocket can choose between two protocols, [`v1.bin.spacetimedb`](#protocols-binary-protocol) and [`v1.text.spacetimedb`](#protocols-text-protocol). Clients should include one of these protocols in the `Sec-WebSocket-Protocol` header of their request.
 
-| `Sec-WebSocket-Protocol` header value | Selected protocol          |
-| ------------------------------------- | -------------------------- |
-| `v1.bin.spacetimedb`                  | [Binary](#binary-protocol) |
-| `v1.text.spacetimedb`                 | [Text](#text-protocol)     |
+| `Sec-WebSocket-Protocol` header value | Selected protocol                    |
+| ------------------------------------- | ------------------------------------ |
+| `v1.bin.spacetimedb`                  | [Binary](#protocols-binary-protocol) |
+| `v1.text.spacetimedb`                 | [Text](#protocols-text-protocol)     |
 
 ### Binary Protocol
 
-The SpacetimeDB binary WebSocket protocol, `v1.bin.spacetimedb`, encodes messages using [ProtoBuf 3](https://protobuf.dev), and reducer and row data using [BSATN](/docs/bsatn).
+The SpacetimeDB binary WebSocket protocol, `v1.bin.spacetimedb`, encodes messages using [ProtoBuf 3](https://protobuf.dev), and reducer and row data using [BSATN](/docs/data-format/bsatn).
 
 The binary protocol's messages are defined in [`client_api.proto`](https://github.com/clockworklabs/SpacetimeDB/blob/master/crates/client-api-messages/protobuf/client_api.proto).
```

Comment:
This is broken as we don't use protobufs anymore. What should this be replaced with?

Reply:
Ah, we need to rewrite this whole page, pretty much. Perhaps we should replace it with a stub for now, and aim to rewrite it after all the WebSocket API changes we have planned.

Reply:
Noted. Will do so in a separate PR after this